src/direct-eval-code/default/func-decl/a-following-parameter-is-named-arguments.template

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2020 Rick Waldron, André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+path: language/eval-code/direct/func-decl-a-following-parameter-is-named-arguments-
+name: Declare |arguments| when a following parameter is named |arguments|.
+esid: sec-evaldeclarationinstantiation
+flags: [noStrict]
+---*/
+
+
+assert.sameValue("arguments" in this, false, "No global 'arguments' binding");
```

Comment:
```suggestion
assert.sameValue("arguments" in this, false, "No global 'arguments' binding before eval");
```

Reply:
this applies to the other matching templates (40 ish of them)

Reply:
Those assertions are gone because it erroneously penalizes engines for having a global `arguments` object, when there is no rule against that.